webapp/src/components/FavoritesCounter/FavoritesCounter.container.ts

```diff
@@ -1,18 +1,35 @@
 import { connect } from 'react-redux'
+import { Item } from '@dcl/schemas'
+import {
+  pickItemAsFavoriteRequest,
+  unpickItemAsFavoriteRequest
+} from '../../modules/favorites/actions'
+import { getIsPickedByUser, getCount } from '../../modules/favorites/selectors'
 import { RootState } from '../../modules/reducer'
+import { getAddress } from '../../modules/wallet/selectors'
+import FavoritesCounter from './FavoritesCounter'
 import {
   MapDispatch,
   MapDispatchProps,
-  MapStateProps
+  MapStateProps,
+  OwnProps
 } from './FavoritesCounter.types'
-import FavoritesCounter from './FavoritesCounter'
 
-// TOOD: use the values from the store
-const mapState = (_state: RootState): MapStateProps => ({
-  isPickedByUser: Math.floor(Math.random() * 2) > 0,
-  count: Math.floor(Math.random() * 5000)
-})
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  const {
+    item: { id: itemId }
+  } = ownProps
+  return {
+    isPickedByUser: getAddress(state)
+      ? getIsPickedByUser(state, itemId)
+      : false,
```

Comment:
If the user isn't logged in, the pickedByUser property will be false, so there shouldn't be any issues with asking the selector directly for the property.
```suggestion
    isPickedByUser: getIsPickedByUser(state, itemId)
```